wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -352,6 +352,12 @@ public void removeWorkspace(String workspaceId) throws ConflictException, Server
      * @param accountId
      *         account which should be used for this runtime workspace or null when
      *         it should be automatically detected
+     * @param isRestore
+     *         if <code>true</code> workspace will be restored from snapshot if snapshot exists,
+     *         if <code>false</code> workspace will not be restored from snapshot
+     *         even if auto-restore is enabled and snapshot exists,
+     *         if <code>null</code> workspace will be restored from snapshot
+     *         only if auto-restore is enabled and snapshot exists
```

Comment:
AFAIK there are 3 options
- system wide parameter
- workspace attribute
- method parameter

can you please define priority of those in the javadoc.